### PR TITLE
i18n prep: logging: standardize logging callsite conventions (SC-1602)

### PR DESCRIPTION
**Pull request by orndorffgrant**
## Why is this needed?
This commit standardizes our logging calls in a couple ways:
 - log messages should be inline strings at the callsite
 - log string formatting should use lazy % style
 - logs shouldn't share strings with printed messages

In the places where log and print calls shared a string, I separated
them and attempted to make the log string more "log-like" and the
printed string more user-friendly. I also moved the printed string
to the messages module.

## Test Steps
Please review every log call and make sure it follows these conventions now.
Please review the changes in messages.

CI should pass

## Checklist

 - [x] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [n/a] Changes here need to be documented, and this was done in: 

## Does this PR require extra reviews?

 - [ ] Yes
 - [x] No

### CODE REVIEW

**Review by renanrodrigo — APPROVED** [review]
phew that is a true chore
lgtm

File: uaclient/entitlements/livepatch.py [hunk 1]
```diff
@@ -288,9 +288,13 @@ def process_contract_deltas(
...
         if any([process_directives, process_token]):
-            msg = "Updating '{}' on changed directives.".format(self.name)
-            LOG.info(msg)
-            event.info(msg)
+            LOG.info(
+                "New livepatch directives or token. running "
+                "setup_livepatch_config"
+            )
+            event.info(
+                "Updating '{}' on changed directives.".format(self.name)
```
Comment (lucasmoura) [inline uaclient/entitlements/livepatch.py:296]: Shouldn't this be on the `messages` module ?

Comment (lucasmoura) [reply uaclient/entitlements/livepatch.py:296]: Or is this going to be addressed in another PR ?

File: uaclient/yaml.py [hunk 2]
```diff
@@ -8,29 +8,31 @@
...
 
 def safe_load(stream):
     try:
         return yaml.safe_load(stream)
-    except AttributeError:
-        msg = BROKEN_YAML_MODULE.format(path=yaml.__path__).msg
-        LOG.error(msg)
-        print(msg, file=sys.stderr)
+    except AttributeError as e:
+        LOG.exception(e)
```
Comment (lucasmoura) [inline uaclient/yaml.py:21]: Should we log here also a similar message to what we are printing ?

File: uaclient/livepatch.py [hunk 3]
```diff
@@ -136,20 +136,21 @@ def status() -> Optional[LivepatchStatusStatus]:
...
         )
         return None
 
     try:
         status_root = LivepatchStatus.from_dict(status_json)
     except IncorrectTypeError:
         LOG.warning(
-            "canonical-livepatch status returned unexpected "
-            "structure: {}".format(out)
+            "canonical-livepatch status returned unexpected " "structure: %s",
```
[the remark this answers is not in the document]
Comment (orndorffgrant) [reply uaclient/livepatch.py:152]: whoops, yep I'll fix it

File: uaclient/yaml.py (hunk 2 above)
Comment (orndorffgrant) [reply uaclient/yaml.py:21]: We can if you want, but `LOG.exception` will include the error and stacktrace pointing to this line, so I think we'll be able to see what happened in the logs

**Review by lucasmoura — APPROVED** [review]

**Pull request merged**